waspc/src/Wasp/Db/Postgres.hs

```diff
@@ -0,0 +1,33 @@
+module Wasp.Db.Postgres
+  ( defaultDevUser,
+    defaultDevDbName,
+    defaultDevPass,
+    defaultDevPort,
+    connectionUrl,
+    defaultDevConnectionUrl,
+  )
+where
+
+import Text.Printf (printf)
+
+-- | TODO: Write explanation on what "dev" stands for in all these variables.
+-- Explain that the idea is that there are default settings we expect from a development database
+-- and that they are defined here.
+-- Maybe move "dev" stuff into additional module, Wasp.Db.Postgres.Dev ?
+defaultDevUser :: String
```

Comment:
Since the username, password and database name don't matter, I vote we keep it as plain as possible e.g. `user=postgres`, `dbName=postgres` and `password=devpass` :)

Reply:
One reason why I added this "waspUser" and "waspDb" suffixes is that I often find it very confusing when I see something like 
```
postgresql://postgres:postgres@localhost:5432/postgres
```
, I would rather see smth like
```
postgresql://postgresWaspDevUser:postgresWaspDevPass@localhost:5432/postgresWaspDevDb
```

Also, I was thinking that it is better to have "wasp" in names because it makes sure that e.g. we won't by accident connect to a db that was run by smb else and not Wasp without knowing -> because as you said, all "postgress" is so default that maybe they already have such db running from some other project and we by accident connect to it.